### ISSUE TITLE
Navigation labels (General)

### ISSUE THREAD

**Issue by adriannelee** [cds-snc/notification-planning#560]
Issue group: Misc
Visibility: 3
Evaluation: Partially Supports
Success Criteria: 2.4.6: Headings and Labels
Needs design: Content

### Description
Generally the navigation reaching are either unlabelled or not labelled in a way that makes their purpose clear. 

![](https://dl.airtable.com/.attachments/7ad42e482d76f96edf633130ab695136/ec418f66/Capturedecranle2022-03-15a13_40_33.png)

**Comment by amazingphilippe** [comment]
Related finding from testing:

https://airtable.com/appWwAmHwDLtpIyko/tbl38n7ofWYBuezFc/viwBYyaQxOttZ0IAw/recAYwlRcb3Q7daMG?blocks=hide

> GC Notify was difficult to navigate for participants. For a user using a screenreader, the labels weren't descriptive enough (e.g. navigation regions, homepage link). For a screen magnification user, the grey banner on the top nav was too light (but other screen magnification users didn't comment on it)

**Comment by amazingphilippe** [comment]
Do an audit of the current navigation label. Come up with a bilingual naming scheme

**Comment by amazingphilippe** [comment]
Please [add your planning poker estimate](https://app.zenhub.com/workspaces/Notify-Planning-614b3ad91bc2030015ed22f5/issues/cds-snc/notification-planning/560?planning-poker) with ZenHub @andrewleith

**Comment by amazingphilippe** [comment]
[Figma for labels](https://www.figma.com/file/r3GzRBjBDWvL2AwCWq1e1W/Navigating-the-site-%7C-Naviguer-le-site?node-id=7800%3A4588)

**Comment by andrewleith** [comment]
@amazingphilippe  It seems like we still have an unlabelled nav element, just under the "Your account and language" nav:
```
<nav class="hidden md:visible md:flex relative justify-end inline-block text-left w-full text-blue visited:text-blue
    link:text-blue no-underline text-small">
```
Should we have nested navs here?

**Comment by amazingphilippe** [comment]
> Should we have nested navs here?

No! We can make a new PR for this

**Comment by andrewleith** [comment]
New card for remaining issue: https://github.com/cds-snc/notification-planning/issues/776
